client/src/js/field/googlemapsfield.js

```diff
@@ -1,21 +1,21 @@
 const targets = Array.from(document.body.querySelectorAll('input[data-mapsfield=mapsfield]'));
-const latField = document.getElementById('GoogleMapsLatField');
```

Comment:
These are constants. If needed, it can be moved to a querySelectorAll to get an array, but they should not rely on a fieldname as below